src/types/mod.rs

```diff
@@ -283,7 +283,7 @@ pub(crate) mod bytes;
 pub(crate) mod capsule;
 #[cfg(not(Py_LIMITED_API))]
 mod code;
-mod complex;
+pub(crate) mod complex;
```

Comment:
Ooops we missed `PyComplexMethods` from the prelude too, mind adding at the same time please?

Reply:
Good catch! Added it in another commit.